src/components/TodoList/TodoList.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react'
+import { List, Avatar } from 'antd'
+import 'antd/dist/antd.css'
```

Comment:
should be imported in main app file

Reply:
But I'm using those imports inside TodoList and they are necessary IMHO here. Do i have to copy them to main js or delete from current and add  to those one?

Reply:
you have to delete antd.css here in TodoList and add to main index, it will work too, but in this case we also don`t need to write this line in every component file